src/Symfony/Component/Validator/ExecutionContextInterface.php

```diff
@@ -101,9 +99,9 @@
      *
      * @api
      *
-     * @deprecated The parameters $invalidValue, $pluralization and $code are
-     *             deprecated since version 2.5 and will be removed in
-     *             Symfony 3.0.
+     * @deprecated since version 2.5.
```

Comment:
this is wrong. It tells that the whole method is deprecated, which is not the case